docs/advanced_guide/cloud/notebooks/Quickstart/Quickstart.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
`docs/advanced_guide/cloud/notebooks/quickstart/` instead of `docs/advanced_guide/cloud/notebooks/Quickstart/`. Just to keep the paths in snake case.

Reply:
Done.